src/schema/tags.ts

```diff
@@ -76,5 +94,25 @@ export const resolvers: IResolvers<any, Context> = traceResolvers({
         hits: hits.map((x) => ({ name: x.value })),
       };
     },
+    onboardingTags: async (
+      source,
+      args,
+      ctx,
+    ): Promise<GQLTagOnboardingResults> => {
+      const hits = await ctx.getRepository(Keyword).find({
+        select: ['value'],
+        where: {
+          status: 'allow',
+          flags: {
+            onboarding: true,
+          },
+        },
+        order: { value: 'ASC' },
+      });
+
+      return {
+        hits: hits.map((x) => ({ name: x.value })),
```

Comment:
```suggestion
        hits: hits.map((hit) => ({ name: hit.value })),
```

Reply:
I'll do all the API work in this PR I think.